BirthdayBot.py

```diff
@@ -1,21 +1,52 @@
-from config import DiscordBotToken
 import discord
+import csv
+import os
+from config import DiscordBotToken
+from discord.ext import commands
+
+file_path = "DiscordBirthdays.csv"
 
 intents = discord.Intents.default()
 intents.message_content = True
 
-client = discord.Client(intents=intents)
+client = commands.Bot(command_prefix = ".", intents = intents)
+
+if os.stat(file_path).st_size == 0:
+    with open('DiscordBirthdays.csv', "w", newline="") as file:
+        myFile =csv.writer(file)
+
+        myFile.writerow(["DiscordName","Birthday"])
 
 @client.event
 async def on_ready():
     print(f'We have logged in as {client.user}')
 
-@client.event
-async def on_message(message):
-    if message.author == client.user:
-        return
+@client.command()
+async def test(ctx):
+    await ctx.send("Hello World!")
+
+@client.command(name="bday")
```

Comment:
Do we need "bday"

Reply:
Argument specifies function command if it needs to be different than the function name, redundant in this case.